README.md

```diff
@@ -1,2 +1,25 @@
-# PalArt-Gallery
+#PalArt-Gallery
```

Comment:
Always upload each file with the related section and branch. 
like there, you upload the readme.md with a branch and pull-request called contact-us

delete any unrelated files to contact-us